### PR TITLE
Tooltip visual updates

### PR DESCRIPTION
**Pull request by tashatitova**
- Updated brushes (inverse style), corner radius and  padding.
- Removed unnecessary corner radius before contract7 part.

tooltip-light
tooltip-dark
tooltip-hc

### CODE REVIEW

File: dev/ToolTip/ToolTip_rs5_themeresources.xaml [hunk 1]
```diff
@@ -2,52 +2,56 @@
...
+        <Setter Property="Padding" Value="{ThemeResource ToolTipBorderPadding}" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="ToolTip">
                     <ContentPresenter x:Name="LayoutRoot"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         Background="{TemplateBinding Background}"
-                        BackgroundSizing="OuterBorderEdge"
+                        contract7Present:BackgroundSizing="OuterBorderEdge"
```
Comment (marksfoster) [inline dev/ToolTip/ToolTip_rs5_themeresources.xaml:46]: contract7Present:BackgroundSizing="InnerBorderEdge"

**Comment by mdtauk** [comment]
Why was the choice of using an inverted Tooltip reversed?  It did help them stand out better from the rest of the UI

File: dev/ToolTip/ToolTip_rs5_themeresources.xaml [hunk 2]
```diff
@@ -2,52 +2,56 @@
...
+        <Setter Property="Padding" Value="{ThemeResource ToolTipBorderPadding}" />
+        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="ToolTip">
                     <ContentPresenter x:Name="LayoutRoot"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         Background="{TemplateBinding Background}"
-                        BackgroundSizing="OuterBorderEdge"
+                        contract7Present:BackgroundSizing="InnerBorderEdge"
```
Comment (marksfoster) [inline dev/ToolTip/ToolTip_rs5_themeresources.xaml:46]: Maybe make this a TemplateBinding: contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"

With setter above: <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />

Comment (tashatitova) [reply dev/ToolTip/ToolTip_rs5_themeresources.xaml]: good idea

**Comment by tashatitova** [comment]
> Why was the choice of using an inverted Tooltip reversed? It did help them stand out better from the rest of the UI

Legacy tooltips won't be updated to inverse so they won't be uniform.

File: dev/ToolTip/ToolTip_rs5_themeresources.xaml [hunk 3]
```diff
@@ -2,52 +2,59 @@
...
+        <Setter Property="Foreground" Value="{ThemeResource ToolTipForegroundBrush}" />
+        <Setter Property="Background" Value="{ThemeResource ToolTipBackgroundBrush}" />
+        <contract7Present:Setter Property="BackgroundSizing" Value="InnerBorderEdge" />
         <Setter Property="BorderBrush" Value="{ThemeResource ToolTipBorderBrush}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ToolTipBorderThemeThickness}" />
         <Setter Property="FontFamily" Value="{ThemeResource ContentControlThemeFontFamily}" />
         <Setter Property="FontSize" Value="{ThemeResource ToolTipContentThemeFontSize}" />
-        <Setter Property="Padding" Value="{ThemeResource ToolTipBorderThemePadding}" />
-        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource OverlayCornerRadius}" />
+        <Setter Property="Padding" Value="{ThemeResource ToolTipBorderPadding}" />
```
Comment (marksfoster) [inline dev/ToolTip/ToolTip_rs5_themeresources.xaml:40]: StaticResource

**Comment by mdtauk** [comment]
> > Why was the choice of using an inverted Tooltip reversed? It did help them stand out better from the rest of the UI
> 
> Legacy tooltips won't be updated to inverse so they won't be uniform.

Ah, yes that would be a problem.  Shame that the older ones wont be changed.

**Review by marksfoster — APPROVED** [review]

File: dev/ToolTip/ToolTip_rs5_themeresources.xaml [hunk 4]
```diff
@@ -2,52 +2,59 @@
...
             <Setter.Value>
                 <ControlTemplate TargetType="ToolTip">
                     <ContentPresenter x:Name="LayoutRoot"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         Background="{TemplateBinding Background}"
-                        BackgroundSizing="OuterBorderEdge"
+                        contract7Present:BackgroundSizing="{TemplateBinding BackgroundSizing}"
                         BorderBrush="{TemplateBinding BorderBrush}"
-                        MaxWidth="320"
+                        MaxWidth="{TemplateBinding MaxHeight}"
```
Comment (marksfoster) [inline dev/ToolTip/ToolTip_rs5_themeresources.xaml:51]: Should be template bound to MaxWidth: MaxWidth="{TemplateBinding MaxWidth}"

**Pull request merged**